modules/database/src/ioc/db/dbChannel.h

```diff
@@ -74,29 +77,30 @@ typedef struct dbChannel {
  */
 typedef db_field_log* (chPostEventFunc)(void *pvt, dbChannel *chan, db_field_log *pLog);
 
-/* Return values from chFilterIf->parse_* routines: */
+/** Return values from chFilterIf->parse_*- routines: */
```

Comment:
For documentation which only has a brief description, you can you the `//!` comment type to avoid having to input `\brief`